code/datums/action.dm

```diff
@@ -319,6 +319,11 @@
 	button_icon_state = "vortex_recall"
 
 /datum/action/item_action/vortex_recall/IsAvailable()
+	var/turf/current_location = get_turf(target)
+	var/area/current_area = current_location.loc
```

Comment:
`get_area`?

Reply:
changing get_turf to get_area just break things while the current method works perfectly fine unless im missing something

Reply:
this needs to be figured out, you shouldn't have to make an intermediate var for this.

Reply:
voodoo magic man